### PR TITLE
feat: channel liquidity alerts on liquidity page

### PR DESCRIPTION
**Pull request by rolznz**
This PR adds alerts to the liquidity page when something needs to be reviewed by the user so they understand why payments might be failing.

No UI changes when your channels are healthy.

- fix: LDK local balance not including amount in reserve
- show current reserve amount if unmet
- unmet channel reserve alert
- low receiving capacity alert
- unbalanced channel warnings
- fix: new channel order sidebar hint not showing on liquidity page

image

image

image

image

### CODE REVIEW

**Comment by im-adithya** [comment]
Maybe we can also add the ArrowUp and ArrowDown beside the Warning icon to show it is related to Sending and Receiving?

**Comment by rolznz** [comment]
> Maybe we can also add the ArrowUp and ArrowDown beside the Warning icon to show it is related to Sending and Receiving?

I'm not sure this is needed or will help - we already have the channel visualization

File: frontend/src/screens/channels/Channels.tsx [hunk 1]
```diff
@@ -361,6 +367,52 @@ export default function Channels() {
...
+              channel.remoteBalance <
+              (channel.localBalance + channel.remoteBalance) * 0.2
+          ) && (
+            <Alert>
+              <AlertTriangle className="h-4 w-4" />
+              <AlertTitle>Low receiving capacity</AlertTitle>
+              <AlertDescription>
+                You likely won't be able to receive payments until you{" "}
+                <ExternalLink
+                  className="underline"
```
Comment (reneaaron) [inline frontend/src/screens/channels/Channels.tsx:384]: Can we just link to the "Increase Inbound" flow here?

File: frontend/src/screens/channels/Channels.tsx [hunk 2]
```diff
@@ -589,6 +642,16 @@ export default function Channels() {
                       const capacity =
                         channel.localBalance + channel.remoteBalance;
 
+                      let channelWarning = "";
+                      if (channel.localSpendableBalance < capacity * 0.1) {
+                        channelWarning =
+                          "Sending balance low. You may have trouble sending payments through this channel.";
```
Comment (reneaaron) [inline frontend/src/screens/channels/Channels.tsx:648]:
```suggestion
                          "Spending balance low. You may have trouble sending payments through this channel.";
```

**Review by reneaaron — APPROVED** [review]
tACK, great changes!

File: frontend/src/screens/channels/Channels.tsx (hunk 1 above)
Comment (reneaaron) [reply frontend/src/screens/channels/Channels.tsx:384]: Same for the link below to increase spending balance...

**Pull request merged**